libs/core/src/components/pds-tabs/pds-tab/pds-tab.scss

```diff
@@ -173,3 +174,47 @@ pds-tab {
     }
   }
 }
+
+.pds-tabs--pill {
+  --color-background-tab: transparent;
+
+  .pds-tab {
+    background-color: var(--color-background-tab);
+    border: var(--pine-border-width-thin) solid transparent;
+    border-radius: var(--pine-dimension-100);
+    color: var(--color-text-default);
+    height: 34px;
+
+    .pds-tab__content {
+      align-items: center;
+      display: flex;
+      justify-content: center;
+      min-width: 90px;
+      padding-block-end: 0;
+    }
+
+    &.is-active,
+    &[aria-selected='true'] {
+      --color-background-tab: var(--pine-color-white);
+      border-color: var(--pine-color-grey-200);
```

Comment:
Should this be `--pine-color-border-disabled`, not sure but I would have thought -`-pine-color-border` but the color doesn't match up - maybe check with design?

Reply:
noted, it's 300 / border